Portal/src/Datahub.Core/Model/Datahub/OpenDataPublishFile.cs

```diff
@@ -2,20 +2,69 @@
 
 namespace Datahub.Core.Model.Datahub
 {
+    /// <summary>
+    /// Represents a file to be published as part of an open data submission, associated with a specific workspace.
+    /// </summary>
     public class OpenDataPublishFile
     {
+        /// <summary>
+        /// Gets or sets the unique identifier for the open data publish file record.
+        /// </summary>
         public long Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the identifier of the open data submission this file belongs to.
+        /// </summary>
         public long SubmissionId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the purpose or reason for including this file in the open data submission.
```

Comment:
The file purpose in this case is the role of the file in the submission. E.g. TBS Opengov submissions can have Dataset and Guide files, as well as a signed IMSO approval form that doesn't get uploaded.

Reply:
updated the doc with your comment